src/utils/index.ts

```diff
@@ -39,11 +39,11 @@ export function constructRecords(
 export const domainExtensionToNamingServiceName = {
   crypto: NamingServiceName.UNS,
   zil: NamingServiceName.ZNS,
-  eth: NamingServiceName.ENS,
-  luxe: NamingServiceName.ENS,
-  xyz: NamingServiceName.ENS,
-  kred: NamingServiceName.ENS,
-  reverse: NamingServiceName.ENS,
+  eth: 'ENS',
+  luxe: 'ENS',
+  xyz: 'ENS',
+  kred: 'ENS',
+  reverse: 'ENS',
 };
```

Comment:
I think it would be better to remove ENS domains altogether. We aren't handling ENS domains any differently (e.g. we don't throw a specific error) so there's no need to keep track of them. 
`findNamingServiceName` already directs to UNS by default. And if someone passes an ENS domain, uns resolution should figure out that the TLD does not exist and throw an appropriate error.